chromium_src/chrome/installer/DEPS

```diff
@@ -1,4 +1,5 @@
 include_rules = [
+  "+brave/browser/brave_vpn/win/brave_vpn_helper",
   "+brave/browser/brave_vpn/win/brave_vpn_wireguard_service",
   "+brave/components/brave_vpn/browser/connection/ikev2/win",
   "+brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper",
```

Comment:
do we need other brave_vpn-related deps here or we can cleanup those?

Reply:
`brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper` is deleted also. and others are used during the install/unintall.